packages/utils/src/lib/sort-report.integration.test.ts

```diff
@@ -0,0 +1,10 @@
+import { report } from '@code-pushup/models/testing';
+import { scoreReport } from './scoring';
+import { sortReport } from './sort-report';
+
+describe('sortReport', () => {
+  it('should sort the report', () => {
```

Comment:
This description could give more insights on what sorting means.
Also a snapshot is most probably the worst testing API here.
A couple of more cases would also help to understand the test better.

Reply:
The compare functions like `compareAudits`, `compareCategoryAudits` are already covered by unit tests. In this case I've kept the snapshot, but used the smaller mock, and left the description.